docs/.vuepress/configs/plugins/notices.ts

```diff
@@ -26,25 +26,39 @@ export const notices: NoticePluginOptions = {
             confirm: true,
             actions: [actionLatest]
         },
+        {
+            path: "/server/v24.10/",
+            title: "EventStoreDB v24.10",
+            content: "You are reading the documentation for EventStoreDB v24.10 LTS.",
+            confirm: true,
+        },
         {
             path: "/server/v23.10/",
             title: "EventStoreDB v23.10",
-            content: "You are reading the documentation for EventStoreDB v23.10 LTS.",
+            content: "EventStoreDB v23.10 nearing end of life. Upgrade to the latest server version.",
```

Comment:
I don't know that `nearing end of life` is correct for 23.10. It will still be supported until October 2025

Reply:
This was the suggest text from Product (Alexey).